openpype/hosts/equalizer/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .addon import (
+    EqualizerAddon,
+    EQUALIZER_HOST_DIR,
+)
```

Comment:
This I believe is missing:

```suggestion
)


__all__ = [
    "EqualizerAddon",
    "EQUALIZER_HOST_DIR",
]

```